### DISCUSSION TITLE
Windows on ARM

### DISCUSSION THREAD

**Discussion by Bloggzy** [Velocidex/velociraptor#3548] [General]
With the recently announced ARM based (Snapdragon) Windows laptops about to hit the market, what are the options / plans for running / supporting Velociraptor on these new devices?

**Comment by scudette** [comment]
You should be able to build an ARM binary by setting the architecture here 

https://github.com/Velocidex/velociraptor/blob/858e2e4a1c597c9dfd1d0389f84ac2186eb3bae7/magefile.go#L368

Just like the darwin ones.

Currently this wont build some reverse engineered things like windows API specific calls (handles, winobj etc) but we can add those in if there is enough interest and someone is willing to test those.

**Reply by Bloggzy** [reply]
Cool, thanks for the info Mike. 

Its not needed right now, but as these new ARM based Windows laptops make their way out into the real world, I suspect the need for an ARM based build may grow?
